external_libraries/json/json_fwd.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef NLOHMANN_JSON_FWD_HPP
```

Comment:
is this the original file that you downloaded or did you modify it?

Reply:
This is the one in the json repository + the declaration of the iterators (in the original just the json class is forward declared)